src/reconcile/rec_write.c

```diff
@@ -2683,13 +2683,6 @@ __rec_hs_wrapup(WT_SESSION_IMPL *session, WT_RECONCILE *r)
      */
     WT_ERR(__wt_hs_delete_updates(session, r));
 
```

Comment:
I looked into why we had this original loop and it's because previously we opened a lookaside cursor here and were trying to save work, that work is no longer done. You are correct that this loop is redundant.